make/doc.go

```diff
@@ -0,0 +1,6 @@
+package make
+
+import (
+	// alpha-build-machinery contains non-go code used in makefiles
+	_ "github.com/openshift/library-go/alpha-build-machinery"
```

Comment:
That's not sufficient, you also need bindata, see https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/pkg/dependencymagnet/doc.go and similarly to the other PR please name the package `dependencymagnet`.